docs(readme): update nvim-navic repository URL

krmbzds opened the pull request:
Lazy.nvim will complain if there is case-mismatch for a dependency that is defined in multiple places.

utilyre commented:
Thanks! I just changed this plugin's workflow since the last one wasn't very viable, so please rebase towards `main` this time :pray:.

Sorry that it keeps changing, I'm gonna move the plugin towards an stable state and then make PR and issue templates so that it becomes much clearer and easier to contribute.

krmbzds commented:
> Thanks! I just changed this plugin's workflow since the last one wasn't very viable, so please rebase towards `main` this time :pray:.

Switched base.

> Sorry that it keeps changing, I'm gonna move the plugin towards an stable state and then make PR and issue templates so that it becomes much clearer and easier to contribute.

No worries!  That's a great idea. It will definitely make it easier to contribute.

krmbzds commented:
I noticed a broken link and fixed that as well.

utilyre commented on README.md:
```diff
@@ -265,4 +265,4 @@ several things you should be aware of.
 ## 👥 Contribution
 
-See [Code of Conduct](/CODEOFCONDUCT.md).
+See [Contributing](/CONTRIBUTING.md).
```
```suggestion
See [Contributing](/CODE_OF_CONDUCT.md).
```

Thanks for pointing out, but it should be like this.

krmbzds replied:
Here's an example `CODE_OF_CONDUCT.md` file: https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md

An example `CONTRIBUTING.md` file:
https://github.com/github/docs/blob/main/CONTRIBUTING.md

Based on it's content, it looks like the file should be renamed to `CONTRIBUTING.md`.

utilyre replied:
Oh thanks, I thought they're the same.

utilyre commented:
Have you rebased towards main though? There still seems to be some lines from `dev`.

krmbzds commented:
> Have you rebased towards main though? There still seems to be some lines from `dev`.

Sorry about that, it should be fine now. :+1:
